blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -244,6 +244,9 @@ def generate(self):
         show_generic_connect = not device_uuids
         for uuid in device_uuids:
             service_uuid = ServiceUUID(uuid)
+            if service_uuid.short_uuid is None:
```

Comment:
If I get it right, this is true for all UUIDs where `service_uuid.reserved` is false. This means it also catches and thus breaks the handling of 03b80e5a-ede8-4b33-a751-6ce34ec4c700 for BLE devices.

The actual problem in my eyes is that `break` in the `elif` branch below is only right if `uuid == '03b80e5a-ede8-4b33-a751-6ce34ec4c700'`. We must not break the loop as long as `show_generic_connect` is `False`.

Reply:
Good one, pushed a new commit for this.